src/connection.hpp

```diff
@@ -69,7 +72,8 @@ class connection
   RequestHandler* default_handler_;
 
   /// Buffer for incoming data.
-  std::array<char, 1024> buffer_;
+  enum { BUF_SIZE = 8192 };
```

Comment:
does this necessarily have to be in an enum? I think we could just use a const int or something like that